prac_09/unreliable_car.py

```diff
@@ -0,0 +1,19 @@
+import random
```

Comment:
In my very short experience, I found it helpful to specify exactly what class I am using from a module. For readability purposes. If I am just importing one or two classes or methods

For short codes it's alright but for long lines of code specifying what exactly you imported helps other coders or future you to get the gist of what's going on when you scan the imports section